homeassistant/components/knx.py

```diff
@@ -26,6 +28,10 @@
 CONF_KNX_FIRE_EVENT = "fire_event"
 CONF_KNX_FIRE_EVENT_FILTER = "fire_event_filter"
 CONF_KNX_STATE_UPDATER = "state_updater"
+CONF_KNX_EXPOSE = "expose"
+CONF_KNX_EXPOSE_TYPE = "type"
+CONF_KNX_EXPOSE_ENTITY_ID = "entity_id"
```

Comment:
Maybe use the existing entity_id constant from const.py?

Reply:
fixed